src/test/java/io/nats/client/impl/ReconnectTests.java

```diff
@@ -774,23 +798,24 @@ public void testForceReconnectQueueBehaviorCheck() throws Exception {
 
             String subject = subject();
             ForceReconnectQueueCheckDataPort.WRITE_CHECK = "PUB " + subject;
-            _testForceReconnectQueueCheck(subject, pubCount, subscribeTime, port, false, 0);
+            _testForceReconnectQueueCheck(subject, pubCount, subscribeTime, port, false, true, 0);
```

Comment:
Did not cover all possible scenarios since in that case locally test was running 17 secs and almost never passed